gallery/src/PageHeader/PageHeader.tsx

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+import packageJson from '../../package.json';
+
+const plaidVillain = require('./plaid-villain.svg');
+const tartanLogotype = require('./tartan-logotype.svg');
+
+import './PageHeader.scss';
+
+const PageHeader = () => (
+  <header className="nx-gallery-page-header">
+    <div className="nx-gallery-page-header__inner">
+      <h1 className="nx-gallery-page-header__logo">
+        <a className="nx-gallery-page-header__home-link"
+           href="#"
+           aria-label="home">
+          <img className="nx-gallery-page-header__logo-image"
+               src={plaidVillain}
+               alt="A cute but villainous Tartan logo" />
+          <img className="nx-gallery-page-header__logotype-image"
+               src={tartanLogotype}
+               alt="Sonatype Tartan" />
+          <span className="nx-gallery-page-header__name">
+            React Shared Component Library
+          </span>
+        </a>
+      </h1>
+      <div className="nx-gallery-page-header__version">
```

Comment:
There is an intention that this text be baseline-aligned with the "React Shared Component Library" text.  I think it would be a lot simpler to tie them together in that way if these two text elements were siblings rather than having the name within the `.nx-gallery-page-header__logo`.

Reply:
> There is an intention that this text be baseline-aligned with the "React Shared Component Library" text. I think it would be a lot simpler to tie them together in that way if these two text elements were siblings rather than having the name within the `.nx-gallery-page-header__logo`.

Yeah it was difficult before because `name` was inside of the home-link and the `version` is not. But that's been changed here: c400966dcd9ada6ecae2cc9d764dcb3363e9fe22